pkg/sass-parser/lib/src/statement/index.ts

```diff
@@ -195,6 +206,15 @@ const visitor = sassInternal.createStatementVisitor<Statement>({
   visitForRule: inner => new ForRule(undefined, inner),
   visitForwardRule: inner => new ForwardRule(undefined, inner),
   visitFunctionRule: inner => new FunctionRule(undefined, inner),
+  visitIfRule: inner => {
+    const rules: Statement[] = [new IfRule(undefined, inner)];
+    for (let i = 1; i < inner.clauses.length; i++) {
```

Comment:
I'm slightly confused why the zero index gets skipped, I imagine it's used in `IfRule`, but I don't see `.clauses[0]`.

I think a comment here can be helpful to understand why it goes from 1 to N rather than 0 to N.